vars/isRelease.groovy

```diff
@@ -0,0 +1,30 @@
+def call(Map args) {
+
+    def isRelease = isReleaseBranch()
+
+    if (env.BRANCH_NAME == 'master') {
+
+        def isMaster = true
+
+    } else {
+
+        def isMaster = false
+    }
+
+    if (isRelease == true) {
+
+        if (isMaster == true) {
+
+            return true
+
+        } else {
+
+            return false
+        }
+
+    } else {
+
+        return false
+    }
+
```

Comment:
[Do not precede } with empty line if no code immediately follows](https://github.com/gitmlynch/pr_exercise/blob/master/CONTRIBUTING.md#whitespace)

```suggestion
```